app/config/apiMapping.js

```diff
@@ -193,21 +193,21 @@ var apiMapping = {
       'controller': 'status'
     }
   },
-  RemoteProducts: {
+  RemoteProjects: {
     lazy: true,
     all: {
       'endpoint': '/private/queue',
-      'controller': 'products/remote'
+      'controller': 'projects/remote'
```

Comment:
In line with my server side review these endpoints should be changed to remote-projects